src/test/java/ruslan/shastkiv/bookstore/service/ShoppingCartServiceTest.java

```diff
@@ -87,7 +96,7 @@ public void getShoppingCart_validId_returnShoppingCartDto() {
         ShoppingCartDto shoppingCartDto = createShoppingCartDto(USER_ID, List.of());
 
         when(shoppingCartRepository.findById(USER_ID)).thenReturn(Optional.of(cart));
-        when(shoppingCartMapper.toDto(cart)).thenReturn(shoppingCartDto);
+        //when(shoppingCartMapper.toDto(cart)).thenReturn(shoppingCartDto);
```

Comment:
Do you need this commented code to be present here? Fix the same issue in all places

Reply:
Not fixed